gcampus/tools/templates/gcampustools/components/measurement_kit_card.html

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+<div class="col">
+    <div class="card p-3 mb-3 bg-light border-0">
+        <h1 class="text-primary">
+            {{ kit.short_name|safe }}
+        </h1>
+        <h5 class="text-secondary">
+            {{ kit.name }}
+        </h5>
+        <a href="{% url "gcampustools:od-converter-overview" pk_kit=kit.pk %}" class="stretched-link"></a>
+    </div>
+</div>
```

Comment:
Are you planning on adding more information to these cards? If yes, I like the column like cards as there can be additional information about the kit. If not IMO there is too much empty space in the column cards. Maybe make them half a column.